gradio/utils.py

```diff
@@ -1460,9 +1461,9 @@ def error_payload(
     error: BaseException | None, show_error: bool
 ) -> dict[str, bool | str | float | None]:
     content: dict[str, bool | str | float | None] = {"error": None}
-    show_error = show_error or isinstance(error, Error)
+    show_error = show_error or isinstance(error, (Error, AppError))
     if show_error:
-        if isinstance(error, Error):
+        if isinstance(error, (Error, AppError)):
```

Comment:
Shouldn't it just be AppError if Error inherits from AppError?

Reply:
yup

Reply:
fixed